src/app/store-stats/MaterialCounts.tsx

```diff
@@ -11,10 +15,15 @@ import styles from './MaterialCounts.m.scss';
 const retiredMats = [31293053, 49145143, 1305274547, 2014411539];
 const showMats = spiderMats.filter((m) => !retiredMats.includes(m));
 const goodMats = [2979281381, 4257549984, 3853748946, 4257549985, 3702027555];
-const seasonal = [1425776985, 1776857076];
-
+const seasonal = [747321467];
```

Comment:
```suggestion
const seasonal = [747321467, 178490507];
```
include Risen Umbral Energy, the currency used to focus engrams, here?

Reply:
thanks